docs/codespaces.md

```diff
@@ -19,13 +19,30 @@ You can create and access a codespace either via the browser or by connecting di
 You only need to do the following the first time you set up the codespace.
 
 ### Add Environment Variables
-- Copy the `sample.env` (run commands from within the remote container, i.e. browser or VS Code terminal)
+- Copy the `codespaces.sample.env` (run commands from within the remote container, i.e. browser or VS Code terminal)
 ```bash
-cp sample.env .env
+cp codespaces.sample.env .env
 ```
 - Add `POSTGRES_URL="postgresql://digitalmarketplace:digitalmarketplace@localhost:5432/digitalmarketplace"` to the `.env` file.
 - Add additional desired environment variables to the `.env` file (refer to the [Environment Variables](https://github.com/button-inc/digital_marketplace/blob/main/README.md#environment-variables) section in the README)
 
+### Keycloak setup
```

Comment:
Suggest adding a note about how you'll get a container error when you first start it